src/custom-elements/src/components/date-picker/date-picker.tsx

```diff
@@ -109,6 +109,8 @@ export class DatePicker {
   excludedWeekdaysArray = [];
   excludedDatesArray = [];
 
+  @State() chiDateReady = false;
```

Comment:
Sometimes there's an error related to `this._vm.dates` being undefined in `chi-date` and that happens because `chi-date-picker` is invoking a function that manipulates `.vm` object. And it's not always that `chi-date` is finished mounting before `chi-date-picker` invokes the function, that's always this error sometimes happens. To avoid it, I created this state variable to keep track whether `chi-date` is finished mounting or not.